### PR TITLE
Add typescript definitions for getStorybook()

### PR DESCRIPTION
**Pull request by joscha**
`2.35.3` add `getStorybook()` in 86d5819a7b8843a1331c33eb7bef17bae0d98006, but Typescript definitions were not updated.

Also `configure` was missing altogether, added in 	0e9d7de

Issue:
```
stories.ts(6,23): error TS2339: Property 'getStorybook' does not exist on type 'typeof "./node_modules/@kadira/storybook/config/storybook"'.
```
when using `getStorybook()`.

## What I did

Added TS definition.

## How to test
Run

`test.ts`, e.g.:

```
import * as storybook from '@kadira/storybook';
import './.storybook/config.js';

console.log(storybook.getStorybook());
```
with `tsc`.

### CODE REVIEW

File: config/storybook.d.ts [hunk 1]
```diff
@@ -13,3 +15,15 @@ export function addDecorator(decorator: StoryDecorator): void;
 export function linkTo(name: string, ...params: any[]): void;
 export function storiesOf(name: string, module: any): Story;
 export function action(name: string, ...params: any[]): any;
+
+type StoryObject = {
+  name: string,
+  render: RenderFunction,
+};
+
+type StoryBucket = {
```
Comment (joscha) [inline config/storybook.d.ts]: Not 100% sure about the naming here, happy to take any suggestion.

**Comment by joscha** [comment]
Updated to not overwrite the changes from https://github.com/storybooks/react-storybook/pull/680

**Comment by joscha** [comment]
@ndelangen any chance to get this in if I rebase it again?

**Comment by ndelangen** [comment]
Yeah, I think so!

**Review by ndelangen — APPROVED** [review]

**Comment by joscha** [comment]
👍 @ndelangen rebased!

**Comment by joscha** [comment]
ta!

**Comment by joscha** [comment]
any chance to get this in a release @ndelangen?

**Comment by ndelangen** [comment]
Yes this will get released in 3.0.0, I don't have a date for that yet, should be an alpha release soonish. My guess this at the end of this month or in the beginning of next month.

**Pull request merged**